lib/config/options/index.ts

```diff
@@ -408,23 +408,15 @@ const options: RenovateOptions[] = [
     type: 'boolean',
     default: false,
   },
-  {
-    name: 'forkMode',
-    description:
-      'Set to `true` to fork the source repository and create branches there instead.',
-    stage: 'repository',
-    type: 'boolean',
-    default: false,
-    globalOnly: true,
-  },
   {
     name: 'forkToken',
     description:
-      'Will be used on GitHub when `forkMode` is set to `true` to clone the repositories.',
+      'Configure a personal access token here to enable "fork mode".',
```

Comment:
```suggestion
      'Set a personal access token here to enable "fork mode".',
```

I think `set` is easier than `configure`.